charts/refractr/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: refractr
+description: A Helm chart for Kubernetes
```

Comment:
Here you should add a description of your Chart, like: "A Helm chart for installing refractr into Kubernetes"

Reply:
Done